### PR TITLE
Add additional exporters outside of kube-monitoring as it is currently refactored

### PR DESCRIPTION
**Pull request by SchwarzM**
I did not know the correct location please advise if this looks good to you and we reference these with requirements in the future

### CODE REVIEW

**Comment by auhlig** [comment]
Difficult question. Having them under `system/exporters` makes sense to me.
However, [infra-exporters](https://github.com/sapcc/helm-charts/tree/master/openstack/infra-exporters) might be the indented place for them. @BugRoger?

File: system/exporter/nfs-exporter/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
+{{ if .Values.enabled }}
```
Comment (auhlig) [inline system/exporter/nfs-exporter/templates/deployment.yaml]: `enabled` not mentioned in .Values

Comment (SchwarzM) [reply system/exporter/nfs-exporter/templates/deployment.yaml]: removed the leftover flag it should be enabled / disabled via requirements

**Comment by SchwarzM** [comment]
hmm nas nothing to do with openstack .... X_x

**Pull request merged**